salt/modules/localemod.py

```diff
@@ -8,6 +8,7 @@
 import logging
 import re
 import os
+import dbus
```

Comment:
We need to gate this import and then return False from `__virtual__` if it is not installed.